From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martin Kampas <martin.kampas@jolla.com>
Date: Thu, 8 Sep 2022 10:04:50 +0200
Subject: [PATCH] docs: Ensure build reproducibility

---
 docs/xep.xsl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/xep.xsl b/docs/xep.xsl
index 2b4505ded325a1c53e2106ed9c8f7d036e786ba7..3796d5a2349414d31c3cde50502b92b8609d9531 100644
--- a/docs/xep.xsl
+++ b/docs/xep.xsl
@@ -401,7 +401,7 @@ OR OTHER DEALINGS IN THE SOFTWARE.
         <xsl:value-of select="$object/@id"/>
       </xsl:when>
       <xsl:otherwise>
-        <xsl:value-of select="generate-id($object)"/>
+        <xsl:number count='*' level='any'/>
       </xsl:otherwise>
     </xsl:choose>
   </xsl:template>
