diff --git a/wocky/wocky-xmpp-reader.c b/wocky/wocky-xmpp-reader.c
index f0eaa92..721e93d 100644
--- a/wocky/wocky-xmpp-reader.c
+++ b/wocky/wocky-xmpp-reader.c
@@ -328,6 +328,7 @@ wocky_xmpp_reader_finalize (GObject *object)
   /* free any data held directly by the object here */
   g_queue_free (priv->stanzas);
   g_queue_free (priv->nodes);
+  g_free (priv->default_namespace);
 
   if (priv->error != NULL)
     g_error_free (priv->error);
